portfolio/src/main/webapp/script.js

```diff
@@ -28,103 +28,213 @@ function addRandomGreeting() {
 }
 
 //*********************** LOADING SCREEN ***********************
-function setVisible(selector, bool) {
+
+/**
+ * Sets elements to visible or not.
+ */
+function setHidden(selector, bool) {
     const el = document.getElementById(selector);
-    el.classList.toggle('hiddenclass', bool);
+    el.classList.toggle('hidden', bool);
 }
 
-function swapLoaderForContent() {
-    setVisible('header', false);
-    setVisible('content', false);
-    setVisible('loaderid', true);
+/**
+ * Gets rid of the loading screen and displays the page
+ */
+function swapLoaderForContent(){
+    setHidden('header', false);
+    setHidden('content', false);
+    setHidden('loaderid', true);
 }
 
+//env variables
+let listenNext = null;
+let retrievedInput = null;
+let match_index = 0;
+let matchingNodes = null;
+
 //*********************** SEARCH BAR FUNCTIONS ***********************
-/**
-  * Swaps the loader for the content and enables search
-  */
+
 window.onload = function(){
-    swapLoaderForContent();
+    swapLoaderForContent(); //displays page after loaded
```

Comment:
nit: add a space between // and your comment so:

// displays page after loaded

same with other comments